src/garage/envs/dm_control/dm_control_env.py

```diff
@@ -87,17 +178,19 @@ def render(self, mode='human'):
             ValueError: if mode is not supported.
         """
         # pylint: disable=inconsistent-return-statements
-        if mode == 'human':
-            if not self._viewer:
-                title = 'dm_control {}'.format(self._name)
-                self._viewer = DmControlViewer(title=title)
-                self._viewer.launch(self._env)
-            self._viewer.render()
-            return None
-        elif mode == 'rgb_array':
+        if mode == 'rgb_array':
             return self._env.physics.render()
         else:
-            raise ValueError
+            raise ValueError('Supported render modes are {}, but '
+                             'got render mode {} instead.'.format(
+                                 self.render_modes, mode))
+
+    def visualize(self):
+        """Creates a visualization of the environment."""
+        if not self._viewer:
+            title = 'dm_control {}'.format(self._name)
```

Comment:
i think this new API allows us to delete `DmControlViewer` and just use the built-in one: https://github.com/deepmind/dm_control/tree/master/dm_control/viewer

Reply:
Let's put this in another PR?

Reply:
In that case, please file an issue